automation/util/setup.go

```diff
@@ -170,23 +169,23 @@ func SetupTypeTemplates(ctx context.Context) (setupTypes *SetupTypes, err error)
 	}
 
 	// Create ndbSecret template automation.NDB_SECRET_PATH
-	ndbSecret := &v1.Secret{}
+	ndbSecret := &corev1.Secret{}
```

Comment:
we have both a corev1 and a v1 that use: "k8s.io/api/core/v1". We should eliminate one.

Reply:
Yes, that's what I did. Again, go-static check :)